### PR TITLE
#551 vmagent in neofs-storage testnet image

### PR DESCRIPTION
**Pull request by aprasolova**
Signed-off-by: anastasia prasolova <anastasia@nspcc.ru>

### CODE REVIEW

**Comment by alexvanin** [comment]
Fix commit message according to the style. It can look like this:

```
[#551] Add vmagent in neofs-storage testnet image

Signed-off-by: anastasia prasolova <anastasia@nspcc.ru>
```

File: config/testnet/neofs.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+PUBLIC_COLLECTOR="https://collector.fs.neo.org:8428/api/v1/write"
```
Comment (anatoly-bogatyrev) [inline config/testnet/neofs.sh:3]: Could you please fix the port to 8429

File: config/testnet/neofs.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+set -m
+neofs-node --config /config.yml &
+vmagent -promscrape.config=/prometheus.yml -remoteWrite.url=${PUBLIC_COLLECTOR}
```
Comment (alexvanin) [inline config/testnet/neofs.sh:7]: vmagent can start before neofs-node pprof server is ready to work. In this case I see in logs
```
error   VictoriaMetrics/lib/promscrape/scrapework.go:258        error when scraping "http://127.0.0.1:9090/metrics" from job "neofs-node-user" with labels {instance="127.0.0.1:9090",job="neofs-node-user",monitor="user_metrics"}: error when scraping "http://127.0.0.1:9090/metrics": dial tcp4 127.0.0.1:9090: connect: connection refused; try -enableTCP6 command-line flag if you scrape ipv6 addresses
```

The best solution is to start vmagent in docker-compose, I guess. But script also can sleep for a while.

File: config/testnet/neofs.sh (hunk 1 above)
Comment (aprasolova) [reply config/testnet/neofs.sh]: Done in docker-compose file

File: config/testnet/neofs.sh (hunk 2 above)
Comment (aprasolova) [reply config/testnet/neofs.sh]: Agree. I rolled back changes in Dockerfile and added vmagent service into the docker-compose.yml.

**Review by alexvanin — APPROVED** [review]

**Pull request merged**